src/opendr/perception/semantic_segmentation/bisenet/bisenet_learner.py

```diff
@@ -189,7 +189,7 @@ def infer(self, img):
             raise UserWarning("No model is loaded, cannot run inference. Load a model first using load().")
         self.model.eval()
         predict = self.model(image).argmax(dim=1).squeeze().cpu()
-        heatmap = Heatmap(predict)
+        heatmap = Heatmap(predict.numpy())
```

Comment:
**Note:** Fixing this even though it is not in my package. The PR of `bisenet` did not implement type checking for the `Heatmap` class although only Numpy arrays were expected according to the original comments in the code.